man/dccChecklist.Rd

```diff
@@ -10,7 +10,7 @@ Checklist is a component that encapsulates several checkboxes. The values and la
 }
 
 \usage{
-coreChecklist(id=NULL, options=NULL, values=NULL, className=NULL, style=NULL, inputStyle=NULL, inputClassName=NULL, labelStyle=NULL, labelClassName=NULL, ...)
+dccChecklist(id=NULL, options=NULL, values=NULL, className=NULL, style=NULL, inputStyle=NULL, inputClassName=NULL, labelStyle=NULL, labelClassName=NULL, loading_state=NULL, ...)
```

Comment:
These ones still have `...` at the end, but looks like the components themselves lost that a while back?

Reply:
This has been fixed in plotly/dash#638; when that PR is merged, `...` will only appear when wildcards are supported by the component.

Reply:
The components being merged here were 'transpiled' using the correct code from the aforementioned PR ☝️, so the stray ellipsis has been :hocho:.